src/utilities/services/playlist-service.js

```diff
@@ -0,0 +1,47 @@
+import axios from "axios";
+import { encodedToken } from "../token";
+
+export const addVideoInPlaylist = async (
+  video,
+  playlistId,
+  playlistDispatch
+) => {
+  try {
+    const data = await axios.post(
+      `/api/user/playlists/${playlistId}`,
+      {
+        video,
+      },
+      {
+        headers: { authorization: encodedToken },
+      }
```

Comment:
This can be written as a variable called HEADERS and replace this with it, so you don't have to pass it everytime you call a function

Reply:
okay